libnavigation-util/src/test/java/com/mapbox/navigation/utils/timer/MapboxTimerTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.mapbox.navigation.utils.timer
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.setMain
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+
+class MapboxTimerTest {
```

Comment:
suggest to add one more test when timer is canceled and lambda is not called

Reply:
Good suggestion. I added it.

Reply:
Saw this https://github.com/mapbox/mapbox-navigation-android/pull/2499/commits/f476b49cd1399344df56d3dde03ae03add41616e @cafesilencio and sounds good to me 👍